source/documentation/getting_started/get_started.md

```diff
@@ -6,9 +6,11 @@ Your department, agency or team must have a GOV.UK PaaS account. This account is
 
 Once your department, agency or team has an org account, you will need an individual account. Ask your [org manager](orgs_spaces_users.html#org-manager) to authorise the creation of your individual account.
 
+The GOV.UK PaaS is hosted in two locations, London and Ireland. The GOV.UK PaaS team creates all new orgs on the London-based GOV.UK PaaS by default. If you already have an account on the Ireland-based GOV.UK PaaS and request a new org, we will create this new org and account on the London-based GOV.UK PaaS, and you will have two separate accounts.
+
 To provide you with an account, we need to store some personal data about you. Please see our [privacy notice](https://www.cloud.service.gov.uk/privacy-notice) for details.
 
-Once you have an individual account, you can access the [GOV.UK PaaS admin tool](https://admin.cloud.service.gov.uk/) (requires sign in). This tool allows you to, depending on your user role permissions, view and manage your [orgs](orgs_spaces_users.html#organisations), [spaces](orgs_spaces_users.html#spaces) and [users](orgs_spaces_users.html#users-and-user-roles) without using the command line.
+Once you have an individual account, you can access either the [London-based](http://admin.london.cloud.service.gov.uk/) or the [Ireland-based](http://admin.cloud.service.gov.uk/) GOV.UK PaaS admin tool (requires sign in), depending on where your org is hosted. This tool allows you to, depending on your user role permissions, view and manage your [orgs](orgs_spaces_users.html#organisations), [spaces](orgs_spaces_users.html#spaces) and [users](orgs_spaces_users.html#users-and-user-roles) without using the command line.
```

Comment:
Every `http://admin` link in this PR should be `https://admin`. It happens in multiple places, but I won't flag it each time.